syntax = "proto3";

package apollo_param;

enum OPERATOR
{
  UNKNOW = 0;
  GET = 1;
  SET = 2;
}

message EchoRequest {
  string appid = 1;
  string namespace = 2;
  OPERATOR op = 3;
  string key = 4;
}

message EchoResponse {
  string value = 1;
}

service Example {
  rpc Echo(EchoRequest) returns (EchoResponse);
};